src/window.rs

```diff
@@ -784,6 +801,16 @@ impl Window {
     self.window.set_decorations(decorations)
   }
 
+  /// Change whether or not the window will always be below other windows.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **iOS / Android:** Unsupported.
```

Comment:
```suggestion
  /// - **Windows**: There is no guarantee that the window will be the bottom most but it will try to be.
  /// - **iOS / Android:** Unsupported.
```

Reply:
3f4feba